wiki/Medals/Legacy_beatmap_packs/de.md

```diff
@@ -402,5 +400,9 @@ Beatmaps:
 
 - Die Beatmap [Loituma - Ievan Polkka](https://osu.ppy.sh/beatmapsets/2463), welche im Internet Pack (Volume 1) enthalten war, war berüchtigt dafür, in die Sektion [Approved](/wiki/Beatmap/Category#approved) verschoben worden zu sein, auch wenn sie nur aus einem einzigen Spinner bestand. Obwohl die Beatmap kurze Zeit später [von der Approved-Beatmap-Auflistung entfernt wurde](https://osu.ppy.sh/community/forums/posts/149328), blieben ihre Überreste trotzdem im Internet Pack (Volume 1).
 
+## Einzelnachweise
```

Comment:
This term can not be used for general addendums. You would use it, like wikipedia does, to direct the reader to a source of certain information. 
A more general term that would cover the cases I see in this PR better would be `Verweise`. Though I would appreciate a second opinion.

Reply:
"References" has the same issue. Wikipedia separates these into "notes" and "references", but with footnotes as they are, they're grouped together. Not sure if anything can be done about that, other than perhaps changing the name of the heading depending on whether it contains citations or notes.

Reply:
Personally, I wouldn't advise to use different words depending on the content of the *references*. It unnecessarily overcomplicates things.

However, adapting the Wikipedia syntax here would still be acceptable in my opinion. The [philosophy](https://de.wikipedia.org/wiki/Philosophie) article is an example for using `Einzelnachweise`, even though this section not only lists literature recommendations.

Instead, using `Verweise` or `Referenzen` sounds better to me. `Referenzen` is even closer to the original English word, and they are a "Stelle, auf die verwiesen wird, weil sie (...) Auskunft über [etwas] geben kann"*[1]* (cf. [Duden](https://www.duden.de/rechtschreibung/Referenz)). So in my opinion, `Verweise` or `Referenzen` should be prefered.

*[1]* Translation: a section which gets referenced and has more (detailed) information

Reply:
I clarified on [Discord](https://discord.com/channels/188630481301012481/218677502141399041/980081176893063208) that the heading would be either "References", "Notes" or "Notes and references" depending on the types included (they're all footnotes and appear in the same section).

Would something like "Anmerkungen", "Referenzen" or "Anmerkungen und Referenzen" (google translation) work?

Reply:
That works 👍

Reply:
ok I'll go with `Anmerkungen` for now for these three articles, feel free to readjust later